packages/core/src/oidc/init.ts

```diff
@@ -222,6 +223,19 @@ export default function initOidc(
           { sameSite: 'lax', overwrite: true, httpOnly: false }
         );
 
+        if (params[ExtraParamsKey.GoogleOneTapCredential]) {
+          ctx.cookies.set(
```

Comment:
Should we use the OIDC cookie signing key to sign the newly added cookies?

Reply:
Google CSRF token not signed, align with it.